lib/preload.ts

```diff
@@ -826,12 +846,13 @@ export class Preloader extends EventEmitter {
 					status: 'success',
 				};
 				if (this.commit === 'latest') {
-					const {
-						should_be_running__release: { __id },
-					} = await this.balena.models.application.get(this.appId, {
-						$select: 'should_be_running__release',
-					});
-					releaseFilter.id = __id;
+					const { should_be_running__release } =
+						await this.balena.models.application.get(this.appId, {
+							$select: 'should_be_running__release',
+						});
+					releaseFilter.id = (
```

Comment:
This surfaces that we might need to do a follow-up to add an error if the app is set to track the null release.
I guess though that the CLI might already be checking this...